tools/pubsub2inbox/README.md

```diff
@@ -32,12 +32,15 @@ Out of the box, you'll have the following functionality:
   - [Recommendations and Insights reports](examples/recommendations-example.yaml)
      - From [Recommender API](https://cloud.google.com/recommender/docs/overview).
      - Also see [example with attached spreadsheet](examples/recommendations-example-2.yaml).
-  - [Cloud Monitoring alerts](examples/monitoring-config.yaml)
+  - [Cloud Monitoring alerts](examples/monitoring-alert-config.yaml)
+  - [Cloud Monitoring metrics](examples/cai.yaml)
+  - [Cloud Asset Inventory search](examples/cai.yaml)
```

Comment:
These files are missing in the repo or PR. Do you mean `examples/cai-example.yaml`?

Reply:
Good catch, fixed it from `.yml` -> `.yaml`.